benchmarks/DynamicalODE/Henon-Heiles_energy_conservation_benchmark.jmd

```diff
@@ -18,6 +18,8 @@ In order to separate these two, we will use `iip` for the in-place names and `oo
 ```julia
 using OrdinaryDiffEq, Plots, DiffEqCallbacks
 using SciMLBenchmarks
+using ADTypes
+using DifferentialEquations
```

Comment:
you don't need this

Reply:
Made the changes. Actually I searched some docs. So it pointed to these dependencies